src/dev/jest/config.integration.js

```diff
@@ -0,0 +1,8 @@
+import config from './config';
+
+export default {
+  ...config,
```

Comment:
__note:__ that's the reason I moved to `js` config format.

Reply:
At this point I didn't try to fine tune code coverage settings here

Reply:
I think this is fine to begin with. Maybe we want to make it "more controlled" at some point, by having a shared config, and then the relevant additions in each of these config files.